### PR TITLE
Add Experimental Dark

### LINKED ISSUE #526: Make dark mode... Darker?
## Old version:
image
## New version:
Screenshot 2020-10-18 at 11 50 29 AM

### PR DESCRIPTION
**Pull request by mxmou**
**Resolves**

Resolves #526

**Changes**

Added my dark mode, which I decided to name "Experimental Dark" because it was called *Dark mode (experimental!)* in my userstyle for such a long time.

**Reason for changes**

Because I said I'm going to make it "soon": https://github.com/ScratchAddons/ScratchAddons/issues/526#issuecomment-711300578

**Tests**

Still has some issues for which changes to the addon loader will be needed - this is why it's a draft. I will also need to test compatibility with other addons.

### CODE REVIEW

**Comment by WorldLanguages** [comment]
Nevermind, I saw #700.

File: addons/scratchr2/experimental_dark_compatibility.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+:root {
+  --scratchr2-dropdown-image: url(https://mxmou.github.io/customize-scratch/assets/caret.svg);
```
Comment (WorldLanguages) [inline addons/scratchr2/experimental_dark_compatibility.css:2]: Maybe you could save this SVG in the addon folder? I think `url()` functions in CSS are relative to the CSS file itself, not the page they're injected to, so something like `url(../caret.svg)` should work?

**Review by WorldLanguages — APPROVED** [review]

File: addons/scratchr2/experimental_dark_compatibility.css (hunk 1 above)
Comment (mxmou) [reply addons/scratchr2/experimental_dark_compatibility.css:2]: But theme userstyles are injected as `<style>` elements, so the browser doesn't know which file they came from.

Comment (WorldLanguages) [reply addons/scratchr2/experimental_dark_compatibility.css:2]: Nice catch. In the future, I might add something like `%addon.self.dir%` to the userstyle injector.

Comment (WorldLanguages) [reply addons/scratchr2/experimental_dark_compatibility.css:2]: Moved discussion to #741

**Comment by Hans5958** [comment]
Dude just merged without knowing that all PRs are on hold

**Pull request merged**